webrender/src/render_backend.rs

```diff
@@ -461,6 +461,15 @@ impl RenderBackend {
                     // will cancel rendering the frame.
                     self.notifier.lock().unwrap().as_mut().unwrap().new_frame_ready();
                 }
+                ApiMsg::DebugCommand(option) => {
+                    let msg = ResultMsg::DebugCommand(option);
+                    self.result_tx.send(msg).unwrap();
+                    let notifier = self.notifier.lock();
+                    notifier.unwrap()
+                            .as_mut()
+                            .unwrap()
+                            .new_frame_ready();
```

Comment:
looks like `new_frame_ready` gets abused more. Maybe we should just add a `ping` method to be used both here and for `MemoryPressure`?